docker/build_docker.sh

```diff
@@ -162,14 +162,31 @@ function docker_build {
         pipenv install --deploy # fails if lock is outdated
         PIPENV_YES=yes pipenv run pip freeze > requirements.txt
         echo "Pipfile lock generated requirements.txt: "
-        echo "############ REQUIREMENTS.TXT ###########"
+        echo "############ REQUIREMENTS.TXT ############"
         cat requirements.txt
         echo "##########################################"
         [ ! -f requirements.txt ] && echo "WARNING: requirements.txt does not exist, this is ok if python usage is not intended."
         [ ! -s requirements.txt ] && echo "WARNING: requirements.txt is empty"
         # del_requirements=yes
     fi
 
+    if [ -f "pyproject.toml" -a ! -f "requirements.txt" ]; then
+       if [ ! -f "poetry.lock" ]; then
+            echo "Error: pyproject.toml present without poetry.lock. Make sure to commit your poetry.lock file"
+            return 1
+        fi
+
+      curl -sSL https://install.python-poetry.org | python3 -  # download poetry
```

Comment:
as discussed we're gonna move this to dockerbuild image

Reply:
https://github.com/demisto/dockerfiles/pull/10296